docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -24,10 +24,239 @@ Then, install the Amplify UI React package:
 
 <InstallScripts framework="react" />
 
+### Usage
+
+<Tabs>
+  <TabItem title="Next.js 13.4+">
+  <Alert variation="info" role="none">
+    Amplify UI components only work in Client Components. If you need to load them into a Server Component, you need to convert them into client-side components by adding a `'use client'` at the top of your file. Please see instructions below on how you could achieve this. For more info, visit Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages).
+  </Alert>
+#### UI Components
+
+##### Single Component
+
+To use a Amplify UI component in Server Components, you must re-export them as Client Components.
+
+An example below is shown for using Amplify UI `Button` component:
+
+First, create a Client Component that exports the Amplify component.  For example, create`AmplifyUIComponents.jsx` placing `'use client'` at the top , import the component and styles and re-export `Button` component.
```

Comment:
Couple things here:
- prefer italicizing file names to give a create delineation between code itself and file names, same feedback for entire PR
- missing a space here: "create`AmplifyUIComponents.jsx`" 
```suggestion
First, create a Client Component that exports the Amplify component.  For example, create _AmplifyUIComponents.jsx_ placing `'use client'` at the top , import the component and styles and re-export `Button` component.
```